applications/welcome/models/tables.py

```diff
@@ -45,3 +45,60 @@
                 Field('name'),  # Name of variable
                 Field('value', 'text'),  # Json, short please
                 )
+
+#########################################################################
+#These correspond to authentication tables.
+#########################################################################
+
+from gluon.tools import Auth, Service, PluginManager
+
+auth = Auth(db)
+
+## after auth = Auth(db)
+
+db.define_table(
+    auth.settings.table_user_name,
+    Field('first_name', length=128, default=''),
```

Comment:
In web2py there is a way of adding extra fields to the auth table.  Perhaps we should be doing that instead?